src/test/java/uk/gov/dhsc/htbhf/utils/WireMockManagerImpl.java

```diff
@@ -71,7 +71,7 @@ public void setupClaimantServiceMappingsForScenario(ClaimScenario scenario) {
 
     @Override
     public void verifyClaimantServiceRequestMatching(String expectedBody) {
```

Comment:
Should we rename this method to state that only one call was made as we're now asserting that.

Reply:
Good plan: `verifySingleClaimantServiceRequestMatching`?